2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_setup.py:_flush():76] Current SDK version is 0.15.8
2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_setup.py:_flush():76] Configure stats pid to 32561
2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/.config/wandb/settings
2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/universal_manipulation_interface/wandb/settings
2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program': '/home/ubuntu/universal_manipulation_interface/train.py'}
2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_setup.py:_flush():76] Applying login settings: {'mode': 'offline'}
2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_init.py:_log_setup():507] Logging user logs to /home/ubuntu/universal_manipulation_interface/wandb/run-20250709_002232-fcacadrt/logs/debug.log
2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_init.py:_log_setup():508] Logging internal logs to /home/ubuntu/universal_manipulation_interface/wandb/run-20250709_002232-fcacadrt/logs/debug-internal.log
2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_init.py:init():547] calling init triggers
2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2025-07-09 00:22:32,911 INFO    MainThread:32561 [wandb_init.py:init():596] starting backend
2025-07-09 00:22:32,912 INFO    MainThread:32561 [wandb_init.py:init():600] setting up manager
2025-07-09 00:22:32,913 INFO    MainThread:32561 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-07-09 00:22:32,915 INFO    MainThread:32561 [wandb_init.py:init():606] backend started and connected
2025-07-09 00:22:32,920 INFO    MainThread:32561 [wandb_init.py:init():697] updated telemetry
2025-07-09 00:22:32,972 INFO    MainThread:32561 [wandb_init.py:init():730] communicating run to backend with 60.0 second timeout
2025-07-09 00:23:32,998 ERROR   MainThread:32561 [wandb_init.py:init():756] encountered error: Run initialization has timed out after 60.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2025-07-09 00:23:33,817 ERROR   MainThread:32561 [wandb_init.py:init():1165] Run initialization has timed out after 60.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
Traceback (most recent call last):
  File "/home/ubuntu/miniforge3/envs/umi/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1147, in init
    run = wi.init()
  File "/home/ubuntu/miniforge3/envs/umi/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 762, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 60.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2025-07-09 00:23:41,546 WARNING MsgRouterThr:32561 [router.py:message_loop():77] message_loop has been closed
